lib/app.js

```diff
@@ -212,11 +219,11 @@ function extractFields (parsedData) {
 //   };
 
 // }
-// function descriptionText (description) {
-//   // preserve the line breaks when converting to text
-//   const html = '<div>' + description.html().replace(/<\/p>/g, '\n</p>') + '</div>';
-//   return cheerio.load(html)('div').text();
-// }
+function descriptionText (description) {
+  // preserve the line breaks when converting to text
+  const html = cheerio.load('<div>' + description.replace(/\<br\>/g, '\r\n') + '</div>');
+  return cheerio.text(html('div'));
```

Comment:
I think `html.text()` should do here

Reply:
It doesn't

This is a example according to cheerio's doc:

```
const $ = cheerio.load('<div>This is <em>content</em>.</div>')
cheerio.text($('div'))
//=> This is content.
```

Reply:
ok